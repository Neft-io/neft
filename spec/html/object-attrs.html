<div data="{a: 1, b: 2}"></div>