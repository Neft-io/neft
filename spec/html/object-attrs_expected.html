<div data="[object Object]"></div>